VectorMath/VectorMath.swift

```diff
@@ -37,73 +37,73 @@ import Foundation
 
 public typealias Scalar = Float
 
-public struct Vector2 {
-    public var x: Scalar
-    public var y: Scalar
+public struct Vector2: Hashable {
+    var x: Scalar
```

Comment:
I think this will make all the struct members internal. The warning about redundant public scope in Xcode 10.2 only applies to members of extensions, not structs.